src/main/webapp/servlet.js

```diff
@@ -56,5 +56,5 @@ const deleteIndices = async() => {
     await fetch(DELETE, {
         method: POST
 });
-
+    alert("Data Deleted"); 
```

Comment:
Ideally, we should alert before deleting.

Reply:
We are confirming with the user before deleting